src/dali_backend.cc

```diff
@@ -26,9 +26,43 @@
 
 #include "triton/backend/backend_model.h"
 #include "triton/backend/backend_model_instance.h"
+#include "src/dali_executor/utils/utils.h"
 
 namespace triton { namespace backend { namespace dali {
 
+struct ModelParameters {
+  explicit ModelParameters(common::TritonJson::Value &model_config)
+  {
+    common::TritonJson::Value params;
+    model_config.MemberAsObject("parameters", &params);
+    GetMember(params, num_threads_key, &num_threads);
+  }
+
+ private:
+  template <typename T>
+  void GetMember(
+      common::TritonJson::Value &params,
+      const char* key,
+      T *value)
+  {
+    if (params.Find(key)) {
+      common::TritonJson::Value param;
+      TRITON_CALL_GUARD(
+        params.MemberAsObject(key, &param));
+      std::string string_value;
+      TRITON_CALL_GUARD(
+        param.MemberAsString("string_value", &string_value));
```

Comment:
Maybe `"string_value"` should be an argument of this `GetMember` function? Or do you plan to name it like that no matter what?

Reply:
The whole "parameters" section in this config is defined in Protobuff as `map<string, ModelParameter>` and ModelParameters is: 
```
message ModelParameter
{
  string string_value = 1;
}
```
So no matter the specific parameter there will always be this `string_value` field.

Reply:
Sure